ext/dom/tests/modern/common/template_rename.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+<template> element renaming
+--EXTENSIONS--
```

Comment:
You don't seem to be renamig the element in this test? But changing the namespace?

Reply:
The first argument of rename is the desired namespace, and the second argument is the desired name. I keep the namespace the same but change the name.

Reply:
Ah okay.